modules/ROOT/pages/amazon-sns-connector.adoc

```diff
@@ -2,30 +2,21 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: anypoint studio, connector, sns, amazon sns, user guide
 
-[[intro]]
-== Introduction
+Support Category: Select
 
-The Anypoint Connector for Amazon SNS provides connectivity to the Amazon Simple Notification Service (Amazon SNS) API, enabling you to build distributed web-enabled applications. Mule applications can use the Amazon SNS Connector to easily push real-time notification messages to interested subscribers over multiple delivery protocols. MuleSoft maintains this connector under the xref:anypoint-connectors.adoc#connector-categories[_Select_] support policy.
+The Anypoint Connector for Amazon SNS provides connectivity to the Amazon Simple Notification Service (Amazon SNS) API, enabling you to build distributed web-enabled applications. Mule applications can use the Amazon SNS Connector to easily push real-time notification messages to interested subscribers over multiple delivery protocols.
 
-*Release Notes:* xref:release-notes::connector/amazon-sns-connector-release-notes.adoc[Amazon SNS Connector Release Notes]
-
-[[prerequisites]]
 == Prerequisites
 
 To be able to use the Amazon SNS Connector, you must have the following:
 
 * Access to Amazon Web Services - SNS.
-* To access AWS with the connector, you will need the credentials in the form of IAM.
+* AWS Identity and Access Management (IAM) credentials.
 * Anypoint Studio - Enterprise Edition.
 
-[[requirements]]
-=== Hardware and Software Requirements
-
-See xref:hardware-and-software-requirements.adoc[Hardware and Software Requirements].
```

Comment:
Why were the requirements removed?

Reply:
I compared the docs with M4 version to make sure they have comparable contents, and I thought since there are no requirements and release notes, then they are probably not needed here as well.
https://github.com/mulesoft/docs-connectors/blob/latest/modules/ROOT/pages/amazon/amazon-sns-connector.adoc

Reply:
OK, sounds good.